test.sh

```diff
@@ -1,4 +1,20 @@
 #!/bin/sh
 source _virtualenv/bin/activate
 
-PYTHONPATH=$PYTHONPATH:./lib/pymaker:./lib/pyexchange:./lib/ethgasstation-client:./lib/gdax-client py.test --cov=market_maker_keeper --cov-report=term --cov-append tests/
+# Pull the docker image
+docker pull makerdao/testchain-pymaker:unit-testing
```

Comment:
Again, why do we need the MCD Parity container?

Reply:
Good catch, we don't. This was Copypasta. I've removed references to the parity container accordingly.